src/fontra/client/core/glyph-controller.js

```diff
@@ -610,7 +610,9 @@ export class StaticGlyphController {
     return getRepresentation(this, "flattenedPathHitTester");
   }
 
-  getSelectionBounds(selection) {
+  // TODO: Question: It does not feel right to have fontController as an argument here.
+  // But it is needed to get the backgroundImage. Is there a better way?
+  getSelectionBounds(selection, fontController = undefined) {
```

Comment:
Instead of passing the font controller, please pass a function `getBackgroundImageBounds(imageIdentifier)`.

`getBackgroundImageBounds(imageIdentifier)` should become a new method on FontController. We then pass this as follows:
```js
    glyph.getSelectionBounds(selection, fontController.getBackgroundImageBounds.bind(getBackgroundImageBounds));
```

Although I can imagine we can make that last line a little more convenient. On FontController:
```js
    get getBackgroundImageBoundsFunc () {
      return this.getBackgroundImageBounds.bind(this);
```

Reply:
done.